smokes/master.cfg

```diff
@@ -116,5 +116,24 @@ c['db'] = {
     'db_url': "sqlite:///state.sqlite",
 }
 
+
+######## Authorization  part
+authz=util.Authz(
+    allowRules=[util.AnyControlEndpointMatcher(role="admins", defaultDeny=False),
+                util.StopBuildEndpointMatcher(role="integrators"),
+                # *-try groups can start "runtests" builds
+                util.ForceBuildEndpointMatcher(builder="runtests", role="users")],
+    roleMatchers=[util.RolesFromUsername(roles=["admins"], usernames=["homer"]),
+                  util.RolesFromUsername(roles=["integrators"], usernames=["marge"]),
+                  util.RolesFromUsername(roles=["users"], usernames=["bart", "lisa"])]
+)
+
+c['www']['auth']=util.UserPasswordAuth({"homer":"doh!", "marge":"maggie~", "bart":"bulldshit?", "lisa":"Bleeding Gums Murphy"})
+c['www']['authz']=authz
+###################
+# Activation / Deactivation of avatar methods
```

Comment:
please add a comment saying that having default avatar gravatar method causes instabilities due to the fact that tests need to load gravatar icons